src/data-base/GameData.js

```diff
@@ -0,0 +1,24 @@
+const GameModel = require("../model/GameModel");
+const InitialConnect = require("./SequelizeConnection");
```

Comment:
"InitialConnect" is an unclear name, as your file is named "SequelizeConnection", this function could be named "sequelizeConnectionFactory", "createSequelizeConnection", "createDatabaseConnection", etc (remember, function names should be verbs)